pytest_reana/fixtures.py

```diff
@@ -425,7 +425,7 @@ def sample_workflow_workspace(tmp_shared_volume_path):
     """
     test_workspace_path = pkg_resources.resource_filename(
         'pytest_reana',
-        '../tests/test_workspace')
```

Comment:
This was a mistake that was working in my local environment as I had installed pytest-reana in editable mode